src/HumusAmqpModule/Consumer.php

```diff
@@ -263,42 +263,40 @@ public function consume($msgAmount = 0)
     {
         $this->target = $msgAmount;
 
-        foreach ($this->queues as $index => $queue) {
-            if (!$this->timestampLastAck) {
-                $this->timestampLastAck = microtime(1);
-            }
-
-            $message = $queue->get();
+        /** @var \AMQPQueue $queue */
+        foreach($this->queues as $queue) {
+            $queue->consume(function ($message, $queue) {
```

Comment:
First problem here: consume() is a blocking call, so only the first queue gets consumed, not all of them. Any ideas how to improve this? Otherwise I need to check if I can improve things here a little.